src/Avalonia.Base/Threading/DispatcherPriority.cs

```diff
@@ -9,11 +9,7 @@ namespace Avalonia.Threading
     // TODO: These are copied from WPF - many won't apply to Avalonia.
     public enum DispatcherPriority
     {
-        /// <summary>
-        /// The job will not be processed.
-        /// </summary>
-        Inactive = 0,
-
+        MinValue = 1,
```

Comment:
The other members of this enum have doc comments, could we get one for this and `MaxValue` for when we finally manage to get API documentation working?